user/deployment/npm.md

```diff
@@ -132,3 +132,36 @@ after_deploy:
   - ./after_deploy_1.sh
   - ./after_deploy_2.sh
 ```
+
+### Troubleshooting common problems
+
+#### Skipping a deployment with the npm provider because this is not a tagged commit
+
+The most likely cause is that `git push` does not push tags. Be sure to run `git push --tags`.
+
+#### Skipping a deployment with the npm provider because this is not on the required runtime
+
+This is because the node version being used for the test does not match the version specified in the conditonal release section:
+
+```
+language: node_js
+node_js: 6  <--- version used for test
+deploy:
+  ...
+  on:
+    node: node  <--- version condition in deployment
+    tags: true
+    ...
+```
+
+If you are not testing against multiple versions of node, you can remove the `node: ` condition entirely. If you are testing your project against multiple versions of node then you probably added the version condition to the deployment to prevent Travis from attempting to publish the module multiple times in one build. In that case just make sure that the version in `deploy.on.node` is one of the versions listed in `node_js`.
+
+#### npm ERR! You need a paid account to perform this action. For more info, visit: https://www.npmjs.com/private-modules
+
+By default [scoped packages](https://docs.npmjs.com/misc/scope) are not public. If you want to publish your package publicly, you can do so manually by tacking on ` --access public` to the `npm publish` command. However, the best way is to set `access` in the `package.json` file like so:
```

Comment:
By "manually" I meant you can just run "npm publish --access public" on your local workstation to publish the initial version, and then after the first publication I believe the "--access public" is not required for subsequent versions. I read that somewhere but haven't tested it.

I thought it worth mentioning in case changing the package.json file wasn't an available option or something... they could do a manual publish of a pre-release and then re-run the Travis build to publish the correct version. But I can't think of a situation where that would be necessary, so I may cut that part and just reword it to say "One way... " rather than "the best way" because who am I to judge what's best?